IOS/SwiftSeedProject/SwiftSeedProject/Domain/Models/Location.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Location.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 5/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+struct Location {
+    
+    var lat: String
+    var lng: String
```

Comment:
This is a matter of taste.. but most frameworks I've worked with used either Lat/Long or Latitude/Longitude.

'lng' doesn't feel right to me.. I prefer the full names, but in this case, at least add the 'o' between l and n, unless there's any reason not to.